src/components/Payment/BraintreeDropInInterface.vue

```diff
@@ -104,17 +104,26 @@ export default {
 		}
 	},
 	watch: {
-		amount(next, prev) {
-			if (next !== prev) {
-				this.btDropinInstance?.updateConfiguration?.('paypal', 'amount', this.formattedAmount);
-				this.btDropinInstance?.updateConfiguration?.(
-					'googlePay',
-					'transactionInfo',
-					this.googleTransactionInfo
-				);
-				this.btDropinInstance?.updateConfiguration?.('applePay', 'paymentRequest', this.applePaymentRequest);
-			}
-		}
+		// Watch the amount property for changes and update each payment method configuration to ensure correct amount.
+		amount: {
+			handler(next, prev) {
+				if (next !== prev) {
+					this.btDropinInstance?.updateConfiguration?.('paypal', 'amount', this.formattedAmount);
+					this.btDropinInstance?.updateConfiguration?.('paypalCredit', 'amount', this.formattedAmount);
```

Comment:
What is `paypalCredit` specifically compared to `paypal`?

Reply:
Ability to use a paypal credit is part of the paypal modal if you select that option specifically. We don't show a separate button for it though.